admin/developers_tool_kit.php

```diff
@@ -765,9 +765,13 @@ function number_pad_with_spaces($number, $n = 0) {
   <?php require DIR_WS_INCLUDES . 'admin_html_head.php'; ?>
   </head>
   <body>
+    <?php echo $outCount;?>
```

Comment:
Move this under the header, otherwise, it gets 'lost' as it's above the admin menu and any messages displayed for the page.

Reply:
Happy to move. As will only need the one new variable.
Is it still ok if nothing found?

Reply:
If nothing's found, there's a messageStack message indicating as such and the 'header' + 'output' will just show the searching for and then found 0.

Reply:
Excellent. Just checking you were happy.